src/main/java/com/microsoft/sqlserver/jdbc/ISQLServerDataSource.java

```diff
@@ -1066,4 +1066,24 @@
      *        String value for 'maxResultBuffer'
      */
     void setMaxResultBuffer(String maxResultBuffer);
+
+    /**
+     * Sets the number of reconnection attempts if there is a connection failure
+     */
+    void setConnectRetryCount(int count);
+
+    /**
+     * Returns the number of reconnection attempts if there is a connection failure
+     */
+    int getConnectRetryCount();
+
+    /**
+     * Sets the number of seconds between each connection retry attempt
+     */
+    void setConnectRetryInterval(int interval);
+
+    /**
+     * Returns the number of seconds between each connection retry attempt
```

Comment:
NIT:  this is the single-line comment so I think `//` should be used here.

Reply:
javadocs